apps/nsqadmin_to_slack/nsqadmin_to_slack.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+	"net/http"
+
+	"github.com/nsqio/go-nsq"
+	"github.com/nsqio/nsq/internal/version"
+	"github.com/nsqio/nsq/internal/app"
+	"github.com/nsqio/nsq/apps/nsqadmin_to_slack/internal/slack"
+)
+
+
+func main() {
+	topic         := flag.String("topic", "", "NSQ topic")
+	channel       := flag.String("channel", "nsqadmin_to_hipchat", "NSQ channel")
+	slackChannel  := flag.String("slack-channel", "", "Slack channel. i.e. #test")
+	slackToken    := flag.String("slack-token", "", "Slack API Token (may alternately be specified via SLACK_TOKEN environment variable)")
+	maxInFlight   := flag.Int("max-in-flight", 200, "max number of messages to allow in flight")
+	showVersion := flag.Bool("version", false, "print version string")
+	nsqdTCPAddrs     := app.StringArray{}
+	lookupdHTTPAddrs := app.StringArray{}
+	flag.Var(&nsqdTCPAddrs, "nsqd-tcp-address", "nsqd TCP address (may be given multiple times)")
+	flag.Var(&lookupdHTTPAddrs, "lookupd-http-address", "lookupd HTTP address (may be given multiple times)")
+	cfg := nsq.NewConfig()
+	flag.Var(&nsq.ConfigFlag{cfg}, "consumer-opt", "option to passthrough to nsq.Consumer (may be given multiple times, http://godoc.org/github.com/nsqio/go-nsq#Config)")
+
+	flag.Parse()
+
+	if *showVersion {
+		fmt.Printf("nsqadmin_to_hipchat v%s\n", version.Binary)
```

Comment:
@danielhfrank I just noticed this =)

 your [inspiration](https://github.com/danielhfrank/nsqadmin2hipchat) lives on

Reply:
Ha, now that's a trip! Ahead of my time, it seems